translate/src/modules/entitydetails/components/Helpers.tsx

```diff
@@ -32,6 +33,57 @@ type Props = {
   resetContactPerson: () => void;
 };
 
+function MachineryTab() {
+  return (
+    <Tab>
+      <Localized id='entitydetails-Helpers--machinery'>{'MACHINERY'}</Localized>
+      <MachineryCount />
+    </Tab>
+  );
+}
+
+function OtherLocalesTab({ otherlocales }: { otherlocales: LocalesState }) {
+  return (
+    <Tab>
+      <Localized id='entitydetails-Helpers--locales'>{'LOCALES'}</Localized>
+      <OtherLocalesCount otherlocales={otherlocales} />
+    </Tab>
+  );
+}
+
+function TermsTab({
+  terms,
+  isTerminologyProject,
+}: {
+  terms: TermState;
+  isTerminologyProject: boolean;
+}) {
+  if (isTerminologyProject) {
+    return null;
+  }
+  return (
+    <Tab>
+      <Localized id='entitydetails-Helpers--terms'>{'TERMS'}</Localized>
+      <TermCount terms={terms} />
+    </Tab>
+  );
+}
+
+function CommentsTab({
+  teamComments,
+  commentTabRef,
+}: {
+  teamComments: TeamCommentState;
+  commentTabRef: any;
+}) {
+  return (
+    <Tab ref={commentTabRef}>
+      <Localized id='entitydetails-Helpers--comments'>{'COMMENTS'}</Localized>
+      <CommentCount teamComments={teamComments} />
+    </Tab>
+  );
+}
+
```

Comment:
I'd consider defining all of the above inline within the `Helpers()` function body. Then they wouldn't need to get their props passed in separately.